js/replay-embed.js

```diff
@@ -7,8 +7,8 @@ document.write('<script src="https://play.pokemonshowdown.com/js/lib/html-saniti
 document.write('<script src="https://play.pokemonshowdown.com/js/lib/soundmanager2-nodebug-jsmin.js"></script>');
 document.write('<script src="https://play.pokemonshowdown.com/js/config.js?a6"></script>');
 document.write('<script src="https://play.pokemonshowdown.com/js/battledata.js?a6"></script>');
-document.write('<script src="https://play.pokemonshowdown.com/data/pokedex-mini.js?a6"></script>');
-document.write('<script src="https://play.pokemonshowdown.com/data/pokedex-mini-bw.js?a6"></script>');
+document.write('<script src="https://play.pokemonshowdown.com/data/pokedex-mini.min.js?a6"></script>');
+document.write('<script src="https://play.pokemonshowdown.com/data/pokedex-mini-bw.min.js?a6"></script>');
```

Comment:
I don't think that's a valid trade-off, as old-gen data is almost never needed.
However, since the rest of past-gen support is already done differently, it's pointless to keep the load fragmented.
I'll move the merge to a different PR.